sdk/core/core-http/lib/nodeFetchHttpClient.ts

```diff
@@ -6,13 +6,13 @@ import * as http from "http";
 import * as https from "https";
 import "node-fetch";
 
-import { FetchHttpClient } from "./fetchHttpClient";
+import { FetchHttpClient, CommonRequestInfo } from "./fetchHttpClient";
 import { HttpOperationResponse } from "./httpOperationResponse";
 import { WebResource } from "./webResource";
 import { createProxyAgent, ProxyAgent, isUrlHttps } from "./proxyAgent";
 
 interface GlobalWithFetch extends NodeJS.Global {
-  fetch: (input: RequestInfo, init?: RequestInit) => Promise<Response>;
+  fetch: typeof import("node-fetch")["default"];
```

Comment:
huh, this is interesting, I don't think I've seen this style of typing off of imports before